README.md

```diff
@@ -41,23 +41,16 @@ Rune provides a challenge number through `Rune.getChallengeNumber()` that will i
 - Changes your game content every day so to ensure your game is always fresh and entertaining
 - Makes everyone compete under the same conditions to ensure fairness and increase competition
 
-You could e.g. use the challenge number in the following ways:
+Here's two common ways you could use the challenge number:
 
 <details>
-<summary>1) As a seed for randomness</summary>
-&nbsp;
-  
-If your game uses randomness to determine the gameplay (e.g. by randomly generating the maps), you can use the challenge number as the seed to achieve deterministic randomness. For example:
+<summary>1) For deterministic randomness (e.g. map generation)</summary>
+&nbsp; 
 
-```js
-import seedrandom from 'seedrandom' // From https://www.npmjs.com/package/seedrandom
+If your game uses randomness to randomly generate maps, you should use `Rune.deterministicRandom()` instead of `Math.random()`. The random number generator provided by Rune is seeded with the challenge number and will therefore always provide the same random values. In this way, all players will play the same map every time.
```

Comment:
Just a couple suggestions:
- "uses randomness to randomly generate maps" -> "uses randomness to generate maps"?
- "same random values." -> "same random values. When `Rune.gameOver()` is called, the random number generator is reset with the same seed."

Reply:
Also, we say "Here's two common ways you could use the challenge number" up above, but for (1), the dev doesn't actually use the challenge number directly. Maybe we should state that under (1) for clarity?

Reply:
Great suggestions, thanks!

> we say "Here's two common ways you could use the challenge number" up above, but for (1), the dev doesn't actually use the challenge number directly. Maybe we should state that under (1) for clarity?

It currently says "The random number generator provided by Rune is seeded with the challenge number", but can try to make it more clear :)

Reply:
I've tried to update this in https://github.com/rune/rune-games-sdk/pull/39/commits/a23b690e7ba5372412602c2502c288632e46aeec and https://github.com/rune/rune-games-sdk/pull/39/commits/8ce2c14a31fe31b043f2b201fd5766a629dbdcb2.

I didn't include "When Rune.gameOver() is called, the random number generator is reset with the same seed." because I think it might be more of a technical detail. I think it's somewhat captured by "This random number generator will therefore always provide the same random values for the same challenge number" and "all players play the same map".

WDYT?